api/sheet.go

```diff
@@ -227,8 +225,11 @@ type SheetFind struct {
 	Source     *SheetSource
 	Type       *SheetType
 	Payload    *string
-	// If present, will only find sheet which related project containing PrincipalID as an active member.
+	// Find sheets which related project containing PrincipalID as an active member.
```

Comment:
We may want to comment on CreatorID as well because now we have Creator, Principal, Organizer which can be confusing in different use cases. We should mention how they are used depending on sheet visibility.

Reply:
This really complex if I only want to get sheets by one API. I'm thinking about that we can use `GET "/api/sheet/starred"` and `GET "/api/sheet/shared"` to reduce the `GET "/api/sheet"` complexity. wdyt

Reply:
I like it :-)